src/test/java/com/wikia/webdriver/testcases/mobilewikitests/HTMLTitleTests.java

```diff
@@ -36,62 +36,62 @@
   private String[][] testCases = {
       {
           "sktest123",
-          "/wiki/Sktest123_Wiki",
+          "Sktest123_Wiki",
           "Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "sktest123",
-          "/wiki/Style-5H2",
+          "Style-5H2",
           "Style-5H2 | Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "sktest123",
-          "/wiki/TestDisplayTitle",
+          "TestDisplayTitle",
           "testing abc | Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "sktest123",
-          "/wiki/Category:Premium_Videos",
+          "Category:Premium_Videos",
           "Category:Premium Videos | Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "sktest123",
-          "/wiki/Category:Non-premium_Videos",
+          "Category:Non-premium_Videos",
           "Category:Non-premium Videos | Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "sktest123",
-          "/wiki/Category:Premium",
+          "Category:Premium",
           "PremiumVideos | Sktest123 Wiki | FANDOM powered by Wikia"
       },
       {
           "es.pokemon",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "es.pokemon" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)